### PR TITLE
[BPK-3976] Add skip-link docs

### PR DESCRIPTION
**Pull request by georgegillams**
I also added one at the top of the doc page:
Screenshot 2020-12-10 at 14 46 01

Screenshot 2020-12-10 at 14 45 51

Screenshot_2020-12-10 Skip link Backpack

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: docs/src/pages/components/SkipLink/WebSkipLinkPage.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: [
+      <Paragraph>
+        Skip links benefit sighted keyboard or switch users by providing them
+        with a way to skip over lengthy menus or filters. Skip links should be
+        absolute-positioned to ensure that when they become visible they do not
```
Comment (shaundon) [inline docs/src/pages/components/SkipLink/WebSkipLinkPage.js:41]:
```suggestion
        absolutely positioned to ensure that when they become visible they do not
```

No hyphen required and should be 'absolutely' as it's being used as an adjective.

File: docs/src/pages/components/SkipLink/WebSkipLinkPage.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        shift other content.
+        <br />
+        <br />
+        It’s ok for a skip link to overlap other content on the page, as long as
+        you are still able to tab forwards/backwards to that content. The skip
```
Comment (shaundon) [inline docs/src/pages/components/SkipLink/WebSkipLinkPage.js:46]:
```suggestion
        you are still able to tab forwards or backwards to that content. The skip
```

Simpler language

File: docs/src/pages/components/SkipLink/WebSkipLinkPage.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        link is very noticeable, and doesn’t disappear immediately. These are
+        deliberate design decisions to ensure that a user tabbing quickly
+        through the page won’t miss the skip link.
+      </Paragraph>,
```
Comment (shaundon) [inline docs/src/pages/components/SkipLink/WebSkipLinkPage.js:50]: I really like this description, in particular the part near the end explaining the deliberate design decision to pre-empt questions. I have a couple of small suggestions to improve it:

* Not a dealbreaker, but is it worth linking out to a canonical resource about skip links? MDN or Nomensa maybe? It'd be good to give people the opportunity to learn more.
* Can we mention something about this being a desktop-only pattern? I'm assuming it's a desktop-only pattern but I don't know for sure 😅

File: package.json [hunk 4]
```diff
@@ -251,6 +251,7 @@
     "not ie < 9"
   ],
   "dependencies": {
+    "bpk-component-skip-link": "^1.0.1",
```
Comment (shaundon) [inline package.json:254]: Should this be a `devDependency`? It seems odd it's on its own

File: docs/src/pages/components/SkipLink/WebSkipLinkPage.js (hunk 3 above)
Comment (georgegillams) [reply docs/src/pages/components/SkipLink/WebSkipLinkPage.js:50]: I was actually gonna link to [webaim](https://webaim.org/techniques/skipnav/) but forgot to include it 🤦

Comment (georgegillams) [reply docs/src/pages/components/SkipLink/WebSkipLinkPage.js:50]: Nomensa have a good article too!
https://www.nomensa.com/blog/2004/what-are-skip-links

Comment (georgegillams) [reply docs/src/pages/components/SkipLink/WebSkipLinkPage.js:50]: btw skip links are also useful for keyboard/screen-reader users on mobile, so they apply equally.

Comment (shaundon) [reply docs/src/pages/components/SkipLink/WebSkipLinkPage.js:50]: Oh cool, good to know.

**Review by shaundon — APPROVED** [review]

**Pull request merged**